packages/@aws-cdk/aws-codepipeline-actions/lib/manual-approval-action.ts

```diff
@@ -63,8 +65,32 @@ export class ManualApprovalAction extends Action {
     return this._notificationTopic;
   }
 
-  protected bound(scope: Construct, _stage: codepipeline.IStage, options: codepipeline.ActionBindOptions):
-  codepipeline.ActionConfig {
+  /**
+   * grantManualApproval
+   *
+   * https://docs.aws.amazon.com/codepipeline/latest/userguide/approvals-iam-permissions.html
+   *
+   * @param grantable the grantable to attach the permissions to
+   */
+  public grantManualApproval(grantable: iam.IGrantable) {
+    if (!this.stage || !this.stage.pipeline) {
```

Comment:
`stage.pipeline` is never `undefined`, so this condition can be simplified to:

```suggestion
    if (!this.stage) {
```